libs/blocks/email-collection/utils.js

```diff
@@ -265,6 +266,24 @@ export async function getAEPData() {
   }
 }
 
+function waitForModal() {
+  return new Promise((resolve) => {
+    const timeout = setTimeout(() => resolve(), 3000);
+    window.addEventListener('milo:modal:loaded', () => {
+      clearTimeout(timeout);
+      resolve();
+    }, { once: true });
+  });
+}
+
+export async function redirectToSignIn(dialog, redirect = true) {
+  if (!redirect) return;
```

Comment:
I generally prefer having methods as you did it, e.g. have the return condition in the `redirectToSignIn` function.

However given the naming, I feel like "redirectToSignIn(FALSE_DO_NOT_REDIRECT)" is a bit weird... just not calling the method makes more sense, at least to me in this instance 🤷